src/pages/projectCreate/components/ProjectTagsCreateEdit.tsx

```diff
@@ -162,19 +162,19 @@ export const ProjectTagsCreateEdit = ({
     )
   }
 
-  const isDisabled = tags.length >= 3
+  const isDisabled = tags.length >= 4
   const showAddTag = !tagOptions.some((tag) =>
     tag.label.toLowerCase().includes(inputValue.toLowerCase()),
   )
-  const disableShowAddTag = inputValue.length < 3 || createLoading
+  const disableShowAddTag = inputValue.length < 4 || createLoading
 
   return (
     <FieldContainer
       title="Tags"
       subtitle={
         <span>
           {t(
-            'Get discovered more easily by users through Tags. You can select up to 3 project tags.',
+            'Get discovered more easily by users through Tags. You can select up to 4 project tags.',
```

Comment:
please update this text in the english.json file as well, that'll helps us track the updated translations as well.

Reply:
updated